### PR TITLE
Making RSA-SecAnalysis Human readable

### PR DESCRIPTION
**Pull request by maiorrulz**

### CODE REVIEW

**Comment by dorsha** [comment]
@anara123 what is the status of this issue?

**Comment by anara123** [comment]
@dorsha I am on it. There are stuff to be fixed

**Comment by dorsha** [comment]
👍

File: Integrations/integration-RsaNetwitnessSecurityAnalytics.yml [hunk 1]
```diff
@@ -1083,43 +1083,44 @@ script:
...
             break;
         case 'nw-get-components':
             var components = getComponents(sessionId, args);
-            results = { components: components };
+            results = {Type: entryTypes.note, Contents: { components: components }, ContentsFormat: formats.json, HumanReadable: tableToMarkdown(command, components), ReadableContentsFormat: formats.markdown};
             break;
         case 'nw-get-events':
             var events = getEvents(sessionId, args);
-            results = { events: events };
+            results = {Type: entryTypes.note, Contents: { events: events }, ContentsFormat: formats.json, HumanReadable: tableToMarkdown(command, events), ReadableContentsFormat: formats.markdown};
```
Comment (anara123) [inline Integrations/integration-RsaNetwitnessSecurityAnalytics.yml]: You should make the `meta` field human readable (the important data is there).

```
{“format”:“UInt64”,“name”:“sessionid”,“value”:“3”},{“format”:“TimeT”,“name”:“time”,“value”:“1373338230000”},
{“format”:“UInt32”,“name”:“size”,“value”:“3649”},{“format”:“UInt32”,“name”:“payload”,“value”:“1337”},
{“format”:“UInt8”,“name”:“medium”,“value”:“1”},{“format”:“MAC”,“name”:“eth.src”,“value”:“10:BF:48:BA:EF:B8”},
{“format”:“MAC”,“name”:“eth.dst”,“value”:“00:26:CB:32:35:BB”},
{“format”:“UInt16”,“name”:“eth.type”,“value”:“IP”},{“format”:“IPv4”,“name”:“ip.src”,“value”:“192.168.1.98”},
{“format”:“IPv4”,“name”:“ip.dst”,“value”:“168.159.219.138”},{“format”:“UInt8”,“name”:“ip.proto”,“value”:“TCP”},
{“format”:“UInt8”,“name”:“tcp.flags”,“value”:“27”},
```

Comment (anara123) [inline Integrations/integration-RsaNetwitnessSecurityAnalytics.yml]: image

File: Integrations/integration-RsaNetwitnessSecurityAnalytics.yml [hunk 2]
```diff
@@ -1083,43 +1083,44 @@ script:
...
             break;
         case 'nw-get-event-details':
-            results = getEventDetails(sessionId, args);
+            var eventDetails = getEventDetails(sessionId, args);
+            results = {Type: entryTypes.note, Contents: eventDetails, ContentsFormat: formats.json, HumanReadable: tableToMarkdown(command, eventDetails), ReadableContentsFormat: formats.markdown};
```
Comment (anara123) [inline Integrations/integration-RsaNetwitnessSecurityAnalytics.yml]: This is the result when I run this command
image

**Comment by Spivaka** [comment]
@maiorrulz please fix comments and merge

**Pull request closed without merging**